packages/shell-api/src/aggregation-cursor.ts

```diff
@@ -57,7 +57,7 @@ export default class AggregationCursor extends ShellApiClass {
 
   @returnsPromise
   tryNext(): Promise<Document | null> {
-    return (this._cursor as any).tryNext() as any; // TODO: tryNext private, see NODE-2952
+    return this._cursor.tryNext() as any;
```

Comment:
Does thsi still have to be `as any` or will it be `Promise<Document | null>` without the cast?

Reply:
Right, good catch :) Dropped the `as any`